test/Microsoft.AspNetCore.Mvc.RazorPages.Test/Internal/RazorProjectPageRouteModelProviderTest.cs

```diff
@@ -60,18 +49,11 @@ public void OnProvidersExecuting_ReturnsPagesWithPageDirective()
         public void OnProvidersExecuting_AddsPagesUnderAreas()
         {
             // Arrange
-            var fileProvider = new TestFileProvider();
```

Comment:
`TestFileProvider` is pretty fragile - it does exact dictionary lookups and consequently breaks if paths differ by leading or trailing slashes. I ran in to this when working on https://github.com/aspnet/Mvc/issues/7608. We have a first class type that treats paths the correct way, so using these instead.